cfme/infrastructure/datastore.py

```diff
@@ -118,8 +118,7 @@ class DatastoreCollection(Navigatable):
     """Collection class for `cfme.infrastructure.datastore.Datastore`"""
 
     def __init__(self, appliance=None):
-        self.appliance = appliance
-        Navigatable.__init__(self, appliance=self.appliance)
+        Navigatable.__init__(self, appliance=appliance)
```

Comment:
**REQURIED**
According last email from @psav it should be like this:
```python
class DatastoreCollection(NavigatableMixin):
     """Collection class for `cfme.infrastructure.datastore.Datastore`"""
     def __init__(self, appliance):
         self.appliance = appliance
```

Reply:
@quarckster, I agree but I'm not planning to fix collections in my PR. You see this change in my PR only because there was error before and @rbabyuk and me fixed it at the same time.  Her PR was merged first. But she didn't remove self.appliance definition. So, I fixed this small issue.